File: program_sample1.go
package main

import "fmt"
import "time"

func main() {
	t1 := time.Now()
	fmt.Println(t1)
	//	10億回ループで約1.2秒
	for i := 0; i < 1000000000; i++ {
		if i == 5000 {
			fmt.Print(i)
			fmt.Println("");
		}
	}
	t2 := time.Now()
	fmt.Println(t2)
}

File: program_sample1.py
# coding: Shift_JIS
import datetime

d1 = datetime.datetime.today()
print('now:', d1)

# 10億回ループで約150秒
for n in range(1000000000):
	if n == 5000:
		print("5000")

d2 = datetime.datetime.today()
print('now:', d2)
